specs/DataStructures/Account.md

```diff
@@ -24,7 +24,15 @@ Regex for a full account ID, without checking for length:
 ^(([a-z\d]+[\-_])*[a-z\d]+\.)*([a-z\d]+[\-_])*[a-z\d]+$
 ```
 
-There is a rent for the account ID length in case it's less than 11 characters long. See [Economics] for the details.
+There is extra requirements for the amount that account has for the account ID length in case it's less than 11 characters long. See [Economics] for the details.
+
+### Top Level Accounts
+
+Top level accounts are very valuable as they provide root of trust and discoverability for companies, applications and users. To prevent take over by squatters, the top level account names that are shorter than 10 characters they are going to be auctioned off.
+
+Specifically, only `registrar` account can create new top level accounts. `registrar` implements standard Account Naming interface to allow create new accounts.
```

Comment:
Should any top-level account ID be allowed to create if the length is larger than 10 characters?